libs/ledger-live-common/src/families/cosmos/CosmosValidatorsManager.ts

```diff
@@ -1,62 +1,36 @@
 import network from "../../network";
 import { log } from "@ledgerhq/logs";
-import { EnvName, EnvValue, getEnv } from "../../env";
+import { EnvName, EnvValue } from "../../env";
 import { makeLRUCache } from "../../cache";
 import type { CosmosValidatorItem, CosmosRewardsState } from "./types";
 import type { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
-
-// Utils
-const getBaseApiUrl = (currency: CryptoCurrency) => {
-  if (currency.id === "cosmos_testnet") {
-    return getEnv("API_COSMOS_TESTNET_BLOCKCHAIN_EXPLORER_API_ENDPOINT");
-  } else {
-    return getEnv("API_COSMOS_BLOCKCHAIN_EXPLORER_API_ENDPOINT");
-  }
-};
-
-const isStargate = (currency: CryptoCurrency) => {
-  if (currency.id === "cosmos_testnet") {
-    return getEnv("API_COSMOS_TESTNET_NODE") == "STARGATE_NODE";
-  } else {
-    return getEnv("API_COSMOS_NODE") == "STARGATE_NODE";
-  }
-};
-
-const parseUatomStrAsAtomNumber = (uatoms: string) => {
-  return parseFloat(uatoms) / 1000000.0;
-};
+import Crypto from "./crypto/crypto";
 
 export class CosmosValidatorsManager {
-  protected _namespace = "cosmos";
-  protected _version = "v1beta1";
+  protected _version: string;
   protected _currency!: CryptoCurrency;
   protected _minDenom!: string;
   protected _endPoint: EnvValue<EnvName> | undefined;
   protected _rewardsState: any | undefined;
+  private _crypto: Crypto;
```

Comment:
Syntax: variable names don't need the `_` at the beginning.

Reply:
fixed